pkg/server/config.go

```diff
@@ -16,21 +16,30 @@ package server
 
 import (
 	"fmt"
+	"strings"
 
 	"github.com/abcxyz/pkg/cli"
 )
 
 // HandlerConfig defines the set over environment variables required
 // for running this application.
 type HandlerConfig struct {
-	Port           string `env:"PORT,default=8080"`
-	ProjectID      string `env:"PROJECT_ID,required"`
-	SuccessTopicID string `env:"PMAP_SUCCESS_TOPIC_ID,required"`
-	FailureTopicID string `env:"PMAP_FAILURE_TOPIC_ID"`
+	Port                 string `env:"PORT,default=8080"`
+	ProjectID            string `env:"PROJECT_ID,required"`
+	SuccessTopicID       string `env:"PMAP_SUCCESS_TOPIC_ID,required"`
+	FailureTopicID       string `env:"PMAP_FAILURE_TOPIC_ID"`
+	DefaultResourceScope string `env:"PMAP_RESOURCE_SCOPE,required"`
+}
+
+var SupportedResourceScope = map[string]struct{}{
+	"projects":      {},
+	"folders":       {},
+	"organizations": {},
 }
 
 // Validate validates the handler config after load.
 func (cfg *HandlerConfig) Validate() error {
```

Comment:
I wonder how policy_server_test.go passed, as we don't have resource scope there....
We will need two validate, one for mapping - ValidateMappingConfig() and one for policy - ValidatePolicyConfig(), The default resource scope is only needed in mapping.

Reply:
The tests are passed because there are no `success` test cases. We can add the case as a follow up, as we need fake clients for pub/sub, asset, and server.